### ISSUE TITLE
Bug Report - Initial window size failed to account for display scaling for padding

### ISSUE THREAD

**Issue by zypA13510** [microsoft/terminal#4077]
# Environment

```none
Windows build number: [Version 10.0.18362.535]
Windows Terminal version (if applicable): 0.7.3451.0
```

# Steps to reproduce

1. In Windows settings, set display scaling to a value other than 100% (e.g. 150%)
2. In `profiles.json`, set "initialRows" to 80, "initialCols" to 24, and "padding" of default profile to 0
3. Run Windows Terminal, the console size should be 80x22 (2 rows smaller due to #2061)
4. Now in `profiles.json`, set "padding" to 50
5. Run Windows Terminal again, observe the console size
6. Now in `profiles.json`, set "padding" to 100
7. Run Windows Terminal, observe the console size

# Expected behavior

In every case, the console size should be 80x24 (80x22 until #2061 is fixed)

# Actual behavior

When padding is set to 50: 76x20
When padding is set to 100: 72x19
Additionally, if display scaling is set to 100%, the console size is 80x22 in every case.

**Comment by DHowett-MSFT** [comment]
#2061 seems like the master tracking bug for this issue. How is it different?

**Comment by zypA13510** [comment]
> #2061 seems like the master tracking bug for this issue. How is it different?

@DHowett-MSFT if you judge by the title then yes, this is a subset of what's described in #2061, so is anything about having a wrong initial size. However, if you look at the description and most of the comments, #2061 is more about a fixed reduction in `initialRows` (40->37, 27->24, or on my computer, 25->23, 24->22, 22->20, note that the reduction in rows is constant and does not scale with padding), while this issue is more about the relationship between two things: display scaling and padding. It seems that, with higher display scaling, padding takes increasingly more rows and columns space from the initial settings (as you can see from my actual behavior section, 4 rows and 1-2 columns per 50 padding). Also, as I said, this issue does not happen if display scaling is set to 100%, whereas #2061 will happen regardless of display scaling.

So yes, depending on how you look at things, it may be a duplicate. But if you ask me, it's not.

Actually, I believe the cause of this issue could be that #1152 failed to account for the display scaling.

**Comment by zadjii-msft** [comment]
Yea I'm pretty sure this should just be closed as a /dupe of #2061. The _correct_ solution to #2061 will be making sure that both the `IslandWindow` and `NonClientIslandWindow`, make sure to account for padding, tab size, and scrollbar size, when both scaled and unscaled.

Thanks for all the extra details!
